src/components/Pagination/Pagination.tsx

```diff
@@ -4,24 +4,25 @@ import styles from "./Pagination.module.scss";
 
 interface PaginationProps {
   count: number;
-  activePage: number;
+  activePage: string | null;
   handlePaginationChange: (page: number) => any;
 }
 const Pagination: React.FC<PaginationProps> = ({ count, handlePaginationChange, activePage }) => {
   const maxCount = 5;
   const updateCount = (count > maxCount ) ? maxCount : count;
   const paginationArr = Array.from(Array(updateCount), (updateCount, ind) => ind + 1);
   
-  const handleClick = (activePage: number): void => {
-    handlePaginationChange(activePage);
+  const handleClick = (page: number): void => {
+    handlePaginationChange(page);
   };
   
+  // if pages less than 2, Pagination is unnecessary
   if (count < 2) return null;
```

Comment:
it's not good solution
can be not so expected for developer view

let's move it inside  return statement 
`return count < 2 ?  (<ul>....</ul>) : null`